src/clp_ffi_py/ir/PyDecoderBuffer.hpp

```diff
@@ -0,0 +1,183 @@
+#ifndef CLP_FFI_PY_PYDECODERBUFFER_HPP
+#define CLP_FFI_PY_PYDECODERBUFFER_HPP
+
+#include <clp_ffi_py/Python.hpp>  // Must always be included before any other header files
+
+#include <GSL/include/gsl/span>
```

Comment:
Need to add `|gsl` 3rd parties regex in `.clang-format` so it groups it properly.

Also, we can add `GSL/include` to `include_dirs` in `setup.py` so we can write `<gsl/span>` instead of `<GSL/include/gsl/span>`.

Reply:
fixed